ui/components/molecules/M008-Popper/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import createPropTypes from 'json-schema-prop-types'
+import { withStyles, Paper } from '@material-ui/core'
+
+import Popper from '@material-ui/core/Popper'
+
+import schema from './schema'
+import styles from './styles'
+
+const M008 = props => {
+  return (
+    <div>
+      <Popper
+        open
+        placement='bottom'
+      >
+        <Paper className={props.classes.paper} elevation={1}>
```

Comment:
great - i like that paper is a part of this

one thing - can we start following the pattern on pulling the variables out of props at the top of the component. like here: https://github.com/root-systems/cobuy-atomic/blob/master/ui/components/atoms/A015-Select/index.js